bootcd/rpms/genesis_scripts/genesis_scripts.spec

```diff
@@ -11,6 +11,8 @@ Source2:        src/sysconfig-init.diff
 Source3:        src/tty.conf.override
 Source4:        src/genesis-bootloader
 Source5:        src/login-shell
+Source6:        src/ttyS0.conf
+Source7:        src/ttyS1.conf
```

Comment:
They dont. There is an explicit override in /etc/sysconfig/init that specifies only a single TTY should be created (see the diff in the RPM).